lib/openhab/core/items/item.rb

```diff
@@ -28,6 +28,25 @@ def ===(other)
         #   The item's descriptive label.
         #   @return [String, nil]
 
+        #
+        # @!attribute [rw] tags
+        #   The item's tags
+        #   @return [Array<String>]
+        #   @overload tags
+        #     Returns the item's tags.
+        #     @return [Array<String>]
+        #   @overload tags=(*tags)
+        #     Sets the item's tags
+        #     @param [String,Symbol,Semantics::Tag] tags Tags to set.
+        #     @return [void]
+
+        # @!visibility private
+        def tags=(*tags)
+          remove_all_tags
+          tags = DSL::Items::ItemBuilder.normalize_tags(tags.flatten)
```

Comment:
plz to remove the flatten. if someone is passing an array, they should know to do `*tags`